c_src/optics_htable/htable_test.c

```diff
@@ -0,0 +1,130 @@
+#include <stdio.h>
```

Comment:
Were you aware of the optics htable tests and (very simple) benchmarks?
https://github.com/RAttab/optics/blob/master/test/htable_test.c
https://github.com/RAttab/optics/blob/master/test/htable_bench.c

Reply:
Saw htable_test but did not know about htable_bench. I'll make a modified version of them

Reply:
the benches are going to be considerably more complicated so I wouldn't worry too much about those.

Reply:
OK, noted